src/Scaffold_Command.php

```diff
@@ -201,6 +200,106 @@ private function _scaffold( $slug, $assoc_args, $defaults, $subdir, $templates )
 		}
 	}
 
+	/**
+	 * Generate PHP, JS and CSS code for registering a custom block.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <slug>
+	 * : The internal name of the block.
+	 *
+	 * [--title=<title>]
+	 * : The display title for your block.
+	 *
+	 * [--dashicon=<dashicon>]
+	 * : The dashicon to make it easier to identify your block.
+	 *
+	 * [--category=<category>]
+	 * : The category name to help users browse and discover your block.
+	 *
+	 * [--textdomain=<textdomain>]
+	 * : The textdomain to use for the labels.
+	 *
+	 * [--theme]
+	 * : Create files in the active theme directory. Specify a theme
+	 * with `--theme=<theme>` to have the file placed in that theme.
```

Comment:
Argument descriptions are wordwrapped automatically, so no need to wordwrap here.